main.py

```diff
@@ -115,6 +115,14 @@ def get_redirect_route(regex_route, handler, defaults=None):
 
     Warning: this method strips off parameters after the trailing slash. URLs
     with parameters should be formulated without the trailing slash.
+
+    Args:
+        regex_route: string. The route to be redirected.
+        handler: callable or string. Custom handler method to be called.
```

Comment:
make it `function | str` (though I'm not sure whether handler can be a str. can you please check?)

Reply:
Hey @DubeySandeep , I have changed this to function | str, and also checked the docs and saw that the handler can be str too
<img width="1049" alt="Screen Shot 2019-09-22 at 12 55 46" src="https://user-images.githubusercontent.com/26630001/65385527-5c6c0180-dd38-11e9-8ef2-a6b499953e77.png">